test/standalone/Vegetation/test_two_stream.jl

```diff
@@ -67,7 +66,7 @@ py_FAPAR = FT.(test_set[2:end, column_names .== "FAPAR"])
 
         # Compute the predicted FAPAR using the ClimaLSM TwoStream implementation
         K = extinction_coeff(ld[i], θs[i])
-        FAPAR = plant_absorbed_ppfd(RT, FT(1), LAI[i], K, θs[i])
+        FAPAR = plant_absorbed_ppfd(RT, FT(1), LAI[i], K, θs[i], a_soil[i])
```

Comment:
is a_soil an array? (should it be a constant?)

Reply:
These tests just run the TwoStream calculation for a large range of input values and compare to the python TwoStream output for the same values, so it reads the soil albedo from the data file for all of the test cases as an array with one entry for each test case. But this is only specific to these tests it is typically just a constant.

Reply:
thanks!